src/gui/mainwindow.cpp

```diff
@@ -1139,6 +1144,10 @@ void MainWindow::closeEvent(QCloseEvent *e)
     if (!m_forceExit && m_systrayIcon && goToSystrayOnExit && !this->isHidden()) {
         hide();
         e->accept();
+        if (!pref->closeToTrayNotified()) {
+            showNotificationBaloon(tr("qBittorrent is closed to tray"), tr("You won't be reminded again."));
```

Comment:
It would be good to be reminded also of how the user can exit the application and change this behavior.